shimmingtoolbox/cli/b0shim.py

```diff
@@ -1107,15 +1107,19 @@ def max_intensity(fname_input, fname_mask, fname_output, verbose):
         nii_mask = nib.load(fname_mask)
 
     # Shim
-    index_per_slice = shim_max_intensity(nii_input, nii_mask)
+    # Output with 1 index
+    index_per_slice = shim_max_intensity(nii_input, nii_mask) + 1
+
+    # Log the output (1 index)
+    logging.info(f"Max intensity indexes: {index_per_slice}")
```

Comment:
```suggestion
    logger.info(f"Max intensity indexes: {index_per_slice}")
```

Reply:
The index do not appear but the directory does (line 1120).

Reply:
Good catch

Reply:
Still nothing, i tried modifying  line 1120 output but nothing. Does this function get run from fsleyes?

Reply:
Not yet, this is just in ST, I'm waiting for this to be merged to update fsleyes

Reply:
![image](https://user-images.githubusercontent.com/55560824/204655861-333506e6-1ef3-47b4-beff-bc11cacde2ac.png)

Reply:
<img width="1094" alt="Screen Shot 2022-11-29 at 5 05 11 PM" src="https://user-images.githubusercontent.com/47249340/204658499-b98c4e58-fede-4615-836b-fbc1fa065483.png">

Reply:
Peut-être tester avec une deuxième personne qui utilise Linux?

Reply:
We needed to make install